docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Clustering/KMeans.cs

```diff
@@ -51,28 +62,37 @@ public static void Example()
             //   Label: 2, Prediction: 2
 
             // Evaluate the overall metrics
-            var metrics = mlContext.Clustering.Evaluate(transformedTestData, "Label", "Score", "Features");
+            var metrics = mlContext.Clustering.Evaluate(
+                transformedTestData, "Label", "Score", "Features");
+
             PrintMetrics(metrics);
             
             // Expected output:
             //   Normalized Mutual Information: 0.95
             //   Average Distance: 4.17
             //   Davies Bouldin Index: 2.87
 
-            // Get cluster centroids and the number of clusters k from KMeansModelParameters.
+            // Get cluster centroids and the number of clusters k from 
```

Comment:
>k [](start = 64, length = 1)

(k)

Reply:
I think it's clear enough like this.

---
In reply to: [278767220](https://github.com/dotnet/machinelearning/pull/3595#discussion_r278767220) [](ancestors = 278767220)